app/models/employee.rb

```diff
@@ -7,6 +7,17 @@ class Employee < ApplicationRecord
   validates :private_number, numericality: { only_integer: true }
   scope :with_private_number, ->(number) { where(private_number: number) }
 
+  def absences
+    attendances = self.attendances.count
+    month_dates = self.current_month_dates
+    absences = month_dates - attendances
```

Comment:
You don't need to assign the result to a new variable and return it, it's enough to leave the last line like this:
```
month_dates - attendances
```

Reply:
Nice, now that you recommend me this change, can i remember this approach, was studied in the ruby koans.

thanks, DONE.